src/Model/Category.php

```diff
@@ -197,7 +197,7 @@ public function getSort(): int
     }
     
     /**
-     * @param TranslatableAttribute $attribute
+     * @param CategoryAttribute $attribute
      * @return Category
      */
     public function addAttribute(TranslatableAttribute $attribute): Category
```

Comment:
Please change the type to _CategoryAttribute_.

Reply:
The TranslatableAttribute must stay in the argument for backward compatiblity reasons. This is not a problem, because a CategoryAttribute is a TranslatableAttribute, so you still can set an instance of CategoryAttribute here. Same for the other comments.